README.md

```diff
@@ -113,6 +113,26 @@ target MyTarget do
 end
 ```
 
+## Local Development
+
+This project uses [Mise](https://mise.jdx.dev/) and [Tuist](https://tuist.io/) to generate a project for local development. Follow the steps below for the recommended setup for zsh.
+
+```sh
+# install mise
+brew install mise
+# add mise activation line to your zshrc
+echo 'eval "$(mise activate zsh)"' >> ~/.zshrc
```

Comment:
It might be nice if we update this command to add it only if it is not already there. If people naively go through the readme of Blueprint/Workflow/Market/etc they'll end up with 3+ entries in their .zshrc.